--- toggldesktop-7.1.146/src/ui/linux/TogglDesktop/toggl.cpp~	2014-12-04 02:53:08.000000000 +0200
+++ toggldesktop-7.1.146/src/ui/linux/TogglDesktop/toggl.cpp	2014-12-05 15:19:03.383618995 +0200
@@ -179,7 +179,7 @@
 
     QString executablePath = QCoreApplication::applicationDirPath();
     QDir executableDir = QDir(executablePath);
-    QString cacertPath = executableDir.filePath("cacert.pem");
+    QString cacertPath = executableDir.filePath("/etc/certs/ca-certificates.crt");
     toggl_set_cacert_path(ctx, cacertPath.toUtf8().constData());
 
     toggl_on_show_app(ctx, on_display_app);
